package.json

```diff
@@ -44,8 +44,8 @@
     "cheerio": "1.0.0-rc.2",
     "xmldom": "0.1.27",
     "logops": "2.1.0",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master"
+    "iotagent-node-lib": "2.8.1",
+    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#1.1.0"
```

Comment:
I would be better to use just:

```
"lwm2m-node-lib": "1.1.0"
```

However, the package is not yet uploaded to the NPM registry so, in order not delay this any more we use this workaround, which I understand leads to an equivalente result.

NTC (just informative)